app/assets/v2/js/pages/new_bounty.js

```diff
@@ -593,46 +596,48 @@ $(document).ready(function() {
-            gasPrice: gas_price
-          }, function(error, txnId) {
-            indicateMetamaskPopup(true);
-            if (error) {
-              _alert({ message: gettext('Unable to pay bounty fee. Please try again.') }, 'error');
-            } else {
-              ipfsBounty.payload.issuer.address = account;
-              ipfsBounty.payload.fee_tx_id = txnId;
-              ipfsBounty.payload.fee_amount = fee;
-              ipfs.addJson(ipfsBounty, newIpfsCallback);
-              if (typeof ga != 'undefined') {
-                ga('send', 'event', 'new_bounty', 'new_bounty_fee_paid');
-              }
-            }
-          });
+        if (FEE_PERCENTAGE == 0) {
+          deductBountyAmount(fee, '');
         } else {
-          const amountInWei = fee * 1.0 * Math.pow(10, token.decimals);
-          const token_contract = web3.eth.contract(token_abi).at(tokenAddress);
-
-          token_contract.transfer(to_address, amountInWei, { gasPrice: gas_price },
-            function(error, txnId) {
+          if (isETH) {
+            web3.eth.sendTransaction({
+              to: to_address,
+              from: web3.eth.coinbase,
+              value: web3.toWei(fee, 'ether'),
+              gasPrice: gas_price
+            }, function(error, txnId) {
               indicateMetamaskPopup(true);
               if (error) {
                 _alert({ message: gettext('Unable to pay bounty fee. Please try again.') }, 'error');
               } else {
-                ipfsBounty.payload.issuer.address = account;
-                ipfsBounty.payload.fee_tx_id = txnId;
-                ipfsBounty.payload.fee_amount = fee;
-                ipfs.addJson(ipfsBounty, newIpfsCallback);
-                if (typeof ga != 'undefined') {
-                  ga('send', 'event', 'new_bounty', 'new_bounty_fee_paid');
+                deductBountyAmount(fee, txnId);
+              }
+            });
+          } else {
+            const amountInWei = fee * 1.0 * Math.pow(10, token.decimals);
+            const token_contract = web3.eth.contract(token_abi).at(tokenAddress);
+
+            token_contract.transfer(to_address, amountInWei, { gasPrice: gas_price },
+              function(error, txnId) {
+                indicateMetamaskPopup(true);
+                if (error) {
+                  _alert({ message: gettext('Unable to pay bounty fee. Please try again.') }, 'error');
+                } else {
+                  deductBountyAmount(fee, txnId);
                 }
               }
-            }
-          );
+            );
+          }
+        }
+      };
+
+      const deductBountyAmount = function(fee, txnId) {
+        ipfsBounty.payload.issuer.address = account;
+        ipfsBounty.payload.fee_tx_id = txnId;
+        ipfsBounty.payload.fee_amount = fee;
+        ipfs.addJson(ipfsBounty, newIpfsCallback);
+        if (typeof ga != 'undefined') {
+          ga('send', 'event', 'new_bounty', 'new_bounty_fee_paid');
```

Comment:
do we want to still track here? maybe a different event to indicate a zero fee?

Reply:
Added a diff event 🙌  nice catch